### ISSUE TITLE
"Edit this page" link missing for pages

### ISSUE THREAD

**Issue by philwo** [bazelbuild/bazel-website#252]
I wanted to edit this page: https://bazel.build/maintaining/maintainers-guide.html

But it doesn't have an "Edit this page" link, which makes it hard to do so.

In contrast, other sites like e.g. https://docs.bazel.build/versions/3.2.0/bazel-overview.html have that link. Can we add it to the other pages, too?

**Comment by jin** [comment]
Yep, you can do that by adapting this snippet from docs.bazel.build for this website:

https://github.com/bazelbuild/bazel/blob/240cbff44c967dbc9e50f9dbeb99fbd49c82ed5a/site/_layouts/documentation.html#L369-L396

**Comment by philwo** [comment]
Thanks! I'll have a look! 😃 I love this feature.

**Comment by aiuto** [comment]
Why would we have an edit section on the versioned docs?  When we have multiple release tracks going, we would have to edit back in the correct branch, not at head.  This seems potentially error prone unless you can only edit master.

That issue aside, is it really great to have easy edit if we do not have easy preview and dead link testing? I find that unless I can stage the site and examine it I don't trust the edits to the markdown.

**Comment by meisterT** [comment]
I really like the edit links where they are available. If nothing else, they give me a clear signal which source file this was generated from.
